src/Symfony/Component/Form/Event/PostSetDataEvent.php

```diff
@@ -11,7 +11,6 @@
 
 namespace Symfony\Component\Form\Event;
 
-use Symfony\Component\Form\Exception\BadMethodCallException;
```

Comment:
should IMO be added back

Reply:
It's not present in PostSubmitEvent.

Reply:
it is for the next major:

See

// throw new BadMethodCallException('Form data cannot be changed during "form.post_set_data", you should use "form.pre_set_data" instead.');

Reply:
What I mean is that there's the same code in PostSubmitEvent without the import :sweat_smile:

Reply:
Oh I would add the import then 👍

Reply:
I'd follow what fabbot wants on this one, because this will come up again otherwise.
But we should then be careful when merging up to not remove the `use` in 7.0